### PR TITLE
Add CF template that creates ECS cluster.

### PR DESCRIPTION
**Pull request by borovikovd**
Creates a very default EKS cluster assuming default VPC.

### CODE REVIEW

File: deploy/aws/eks/demo-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Amazon EKS Cluster'
+
+Resources:
+  EKSCluster:
+    Type: 'AWS::EKS::Cluster'
+    Properties:
+      Name: 'DemoEKSCluster'
+      RoleArn: 'arn:aws:iam::248441125472:role/eksDemoClusterRole'
```
Comment (vitaly-ivanov) [inline deploy/aws/eks/demo-cluster.yaml]: It's a reference to our AWS account. I think, we should create a new role in this stack.

File: deploy/aws/eks/demo-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Amazon EKS Cluster'
+
+Resources:
+  EKSCluster:
```
Comment (vitaly-ivanov) [inline deploy/aws/eks/demo-cluster.yaml]: Which EC2 instances are used? I don't see where they are provisioned.

File: deploy/aws/eks/demo-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    Type: 'AWS::EKS::Cluster'
+    Properties:
+      Name: 'DemoEKSCluster'
+      RoleArn: 'arn:aws:iam::248441125472:role/eksDemoClusterRole'
+      ResourcesVpcConfig:
+        SubnetIds:
+          - 'subnet-5499442e'
+          - 'subnet-43bb2a2a'
+          - 'subnet-643ada28'
```
Comment (vitaly-ivanov) [inline deploy/aws/eks/demo-cluster.yaml]: Subnets can be created in this stack as well.

**Review by vitaly-ivanov — COMMENTED** [review]
If this stack will be used not only internally, I think more configuration options should be added.

**Comment by borovikovd** [comment]
@vitaly-ivanov I reworked templates, please have a look.

File: README.md [hunk 4]
```diff
@@ -1,41 +1,77 @@
...
+Also, make sure you have AWS CLI installed: [docs](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+
+
+### Create environments in AWS
+
+Before you start, you need a EC2 ssh key. If you don't have one, create one in the aws console and copy it's name:
+
+```bash
+export EC2_SSH_KEY=<NAME>
 ```
```
Comment (vitaly-ivanov) [inline README.md:37]: Do we need ssh to EC2 instances? If not, then the number of required actions can be reduced.

**Review by vitaly-ivanov — APPROVED** [review]
LGTM except for the small comment

File: README.md (hunk 4 above)
Comment (borovikovd) [reply README.md:37]: I'll check if it works without the key during deploy.

**Pull request merged**